### ISSUE TITLE
Windowed Screen Shaking Issue with Simple Video

### ISSUE THREAD

**Issue by Burningocean2012** [visualboyadvance-m/visualboyadvance-m#160]
### Expected behavior
Game to play fine

### Actual behavior
Screen shakes non stop when windowed,stops when fullscreen

### Steps to reproduce the behavior
Open any game with Simple output module while application is windowed

### Crash bugs

If this is a crash bug, and you have the time and inclination, please provide a
symblic backtrace.

Instructions for doing so are
[here](https://github.com/visualboyadvance-m/visualboyadvance-m/blob/master/README.md#reporting-crash-bugs).

### ROM details

Which games are affected by the issue (please state if GBA or GB game):All games

Which games, if any, are NOT affected by the issue (please state if GBA or GB game):

### Option details

Video driver (simple, opengl):Simple

Sound driver (directsound, xaudio, openal, SDL):Audio off(hard of hearing sadly)

### Build details

Operating System (Windows, Mac, Linux (state distribution), etc.):Windows 8.1

If using Linux, specify if using xorg or Wayland:

Version of code (pre-built binary version, or commit ref, or just "master"):2.0.1

### Hardware details:

If your joystick or gamepad isn't working in Linux, please try to troubleshoot
using the information on this page:
https://wiki.archlinux.org/index.php/Gamepad **BEFORE** filing an issue. We use
SDL for joystick support, any problems will generally be an issue with SDL and
**NOT** the emulator.

If relevant, the joystick you are using and the drivers you are using for it:

**Comment by rkitover** [comment]
This doesn't seem like it's related to the app, check your other settings, like in the XBox app which sometimes tries to do stuff to our app.

**Comment by Burningocean2012** [comment]
Hmm,strange.I don't think any Xbox apps are even on since I never use them.Perhaps its some other kind of setting somewhere.Windows can be pretty iffy with pretty much anything.I don't have a problem playing it in OpenGL so my issue isn't really too important.

**Comment by rkitover** [comment]
Oh so you only see the shaking when the video driver is Simple and not OpenGL?

**Comment by Burningocean2012** [comment]
Yes,thats it.Sorry if my description wasn't clear on that.

**Comment by rkitover** [comment]
I'll do some testing to see if I can reproduce this.

**Comment by Burningocean2012** [comment]
Also forot to tell you about filter setting might be involved i guess,turning off filters seems to stop the shaking...

**Comment by Burningocean2012** [comment]
I feel real dumb now,this is a whole different issue involving windows being hot garbage now.I only recently discovered VBA-M even existed.

**Comment by Burningocean2012** [comment]
Do you think i should just remove this issue since my only problem now is different and this was nothing?Sorry if i wasted peoples time.

**Comment by rkitover** [comment]
Yeah it's ok, can you please explain what the window shaking was from? I'm kind of curious.

**Comment by Burningocean2012** [comment]
Well video recording doesn't seem to work but the shaking is the video output not window itself.I really don't know why its doing it.I used CamStudio to record it.Here is a link to video:

http://tinypic.com/player.php?v=64fqlu%3E&s=9

May be hard to really see it since I didn't record in high quaility.

**Comment by ZachBacon** [comment]
I've reopened this because it's still being investigated, and if you have solved the issue, then comment before closing please.

**Comment by Squall-Leonhart** [comment]
Closing this because

* There are no supporting reports (duplicates)
* It doesn't  happen to anyone else
* Report is super old
